css-layout-classes/index.html

```diff
@@ -0,0 +1,93 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>CSS-Layout-Classes</title>
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <div class="container">
+      <div class="row">
+        <div class="column-full">
+          <h1>Choose your character!</h1>
+        </div>
+      </div>
+
+      <div class="row">
+        <div class="column-pic">
+          <img class="pics" src="images/link.png" alt="link" />
+        </div>
+        <div class="column-one-third">
+          <h2>Super Smash Bros. Ultimate</h2>
+          <p>
+            Super Smash Bros. Ultimate received universal acclaim from both
+            critics and players, with some critics calling it the best
+            installment in the series. It received praise for its large amount
+            of content and fine-tuning of existing Smash gameplay elements,
+            although its online mode was widely criticized.
+          </p>
+        </div>
+        <div class="column-pic">
+          <img class="pics" src="images/yoshi.png" alt="yoshi" />
+        </div>
+      </div>
+
+      <div class="row">
+        <div class="column-pic">
+          <img src="images/kirby.png" alt="kirby" />
+        </div>
+        <div class="column-pic">
+          <img src="images/fox.png" alt="fox" />
+        </div>
+        <div class="column-pic">
+          <img src="images/pikachu.png" alt="pikachu" />
+        </div>
+        <div class="column-pic">
+          <img src="images/ness.png" alt="ness" />
+        </div>
```

Comment:
Same thing here. `column-pic` doesn't really tell me anything about how big the columns are. If we want 4 columns, a better class name would be `column-fourth`